src/features/payment/components/PaymentChoice/PaymentMethod.tsx

```diff
@@ -37,21 +37,18 @@ function ImageComponent(method: PaymentInstrumentsType) {
   const [image, setImage] = React.useState<"main" | "alt">("main");
   const onError = React.useCallback(() => setImage("alt"), []);
   const imgSize = { width: "23px", height: "23px" };
-
   const paymentMethodConfig =
     PaymentMethodRoutes[method.paymentTypeCode as PaymentCodeType];
 
   const iconDefault = <DefaultIcon method={method} />;
-
-  if (!paymentMethodConfig?.asset || image !== "main") {
-    return iconDefault;
-  }
-
-  if (typeof paymentMethodConfig?.asset === "string") {
+  if (typeof method?.asset === "string" && image === "main") {
+    const altString = `Logo ${method.name}`;
     return (
       <img
-        src={paymentMethodConfig?.asset}
+        src={method?.asset}
```

Comment:
You could use a constant to memorize method.asset and use it inside the if on line 44 and here on line 48.